apps/wing/src/cli.ts

```diff
@@ -207,7 +207,7 @@ async function main() {
   program
     .command("pack")
     .description("Package the current directory into an npm library (gzipped tarball).")
-    .addOption(new Option("-o --out-file <filename>", "Output filename"))
```

Comment:
This fixes the issue because the interface name doesn't match, right? Could we change the interface instead of this? IMO I like out-file better, so maybe the JS side needs to be outFile.